remme/clients/block_info.py

```diff
@@ -1,8 +1,12 @@
 import logging
+import asyncio
 
 from remme.clients.basic import BasicClient
 from remme.protos.block_info_pb2 import BlockInfo, BlockInfoConfig
+# TODO: Uncomment when proto will be available
```

Comment:
Just update the `protos` submodule, it's already here.

Reply:
Will do